### PR TITLE
IObit.DriverBooster9 version 10.0.0

### PR DESCRIPTION
**Pull request by TheBossMagnus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82467)

### CODE REVIEW

File: manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster9
```
Comment (ItzLevvie) [inline manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: IObit.DriverBooster10
```
You'll also need to change `manifests/i/IObit/DriverBooster9` to `manifests/i/IObit/DriverBooster10` and update the file names as well.

File: manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster9
```
Comment (ItzLevvie) [inline manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.yaml:4]:
```suggestion
PackageIdentifier: IObit.DriverBooster10
```
You'll also need to change `manifests/i/IObit/DriverBooster9` to `manifests/i/IObit/DriverBooster10` and update the file names as well.

File: manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Author: IObit
+PackageName: Driver Booster 9
+PackageUrl: https://www.iobit.com/en/driver-booster.php
+License: Proprietary
+LicenseUrl: https://www.iobit.com/en/eula.php
+Copyright: © 2005 - 2021 IObit. All Rights Reserved
+CopyrightUrl: https://www.iobit.com/en/eula.php
+ShortDescription: Driver Booster 9 Setup
+Description: All-in-one PC optimizer to clean junk files, sweep privacy traces, and boost startup. Speed up with new AI Mode. Smarter & Faster than ever before.
+Moniker: driver-booster-9
```
Comment (ItzLevvie) [inline manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml:20]:
```suggestion
Moniker: driver-booster-10
```

File: manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+PublisherSupportUrl: https://www.iobit.com/en/onlinefeedback.php
+PrivacyUrl: https://www.iobit.com/en/privacy.php
+Author: IObit
+PackageName: Driver Booster 9
+PackageUrl: https://www.iobit.com/en/driver-booster.php
+License: Proprietary
+LicenseUrl: https://www.iobit.com/en/eula.php
+Copyright: © 2005 - 2021 IObit. All Rights Reserved
+CopyrightUrl: https://www.iobit.com/en/eula.php
+ShortDescription: Driver Booster 9 Setup
```
Comment (ItzLevvie) [inline manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Driver Booster 10 Setup
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml (hunk 1 above)
Comment (TheBossMagnus) [reply manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.locale.en-US.yaml]: if someone has v9 it would receive v10 with Winget upgrade?

**Comment by ItzLevvie** [comment]
Close with reason: Fixed with PR 82826 under a new package ID with AppsAndFeatures to allow upgrades from version 9 to version 10 to work;

**Pull request closed without merging**